actors/builtin/miner/miner_test.go

```diff
@@ -3090,6 +3090,11 @@ type actorHarness struct {
 
 	epochRewardSmooth  *smoothing.FilterEstimate
 	epochQAPowerSmooth *smoothing.FilterEstimate
+
+	precommitDealWeight         abi.DealWeight
+	precommitVerifiedDealWeight abi.DealWeight
+	dealWeight                  abi.DealWeight
+	verifiedDealWeight          abi.DealWeight
```

Comment:
I was running into this on #1039 too, but determined we should really make these more easily configurable from the  caller, with a `preCommitConfig` or similar. I shied away from the churn initially, but would prefer we go that path than add these here. You I can do it in my PR.